build.gradle

```diff
@@ -13,6 +13,7 @@ buildscript {
         opensearch_version = System.getProperty("opensearch.version", "2.7.0-SNAPSHOT")
         version_qualifier = System.getProperty("build.version_qualifier", "")
         opensearch_group = "org.opensearch"
+        oldBWCVersion = "2.6.0"
```

Comment:
I didn't find any other way to append a value to existing matrix. So, I'm using this variable as a reference to append the existing opensearch version before version bump to BWC matrix.

Reply:
Is there a case when oldBWCVersion is something else than a minor from `opensearch_version` minus one? E.g.
```opensearch_version = System.getProperty("opensearch.version", "2.7.0-SNAPSHOT")```, we do parse 2.7.0, and major = 2, minor = 7, and  making oldVersion = major + (minor - 1) = 2.6.0. If that's the case then this can be calculated in updateVersion function

Reply:
Ideally it should be same like what you mentioned oldVersion = major + (minor-1) + patch. Let me try to do it that way. Thanks for the suggestion.